### DISCUSSION TITLE
Cleared junk problem

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#296] [Q&A] [answered]
Hi. after last update, my paladin dropped town portal book(and every things) in town and he can't go back to town when he has no potion.
And he try to attack blocked opposite side monster in Pit so he won't move.
Should I roll back?

**Comment by isid0re** [comment]
At this time yes. I'm currently working on the bug fix.

**Comment by isid0re** [comment, marked as answer]
A fix has been pushed to the repo as of this post for the clearjunk.

Attacking is still being fixed but the bot will eventually clear thru the distance and skip those monsters and move on.

**Reply by mirakhany** [reply]
Thank you for update. 
No cleared junk problem now.
but paladin has still path finding problem for now.
